### ISSUE TITLE
Doesn't work for Odoo 13 w/o setting additional ENV variable

### ISSUE THREAD

**Issue by OSevangelist** [Tecnativa/doodba-scaffolding#48]
Changing the major version (in the .env) to Odoo13 to deploy an Odoo13 instance currently results into the following error

```
(INFO) [13:20:01] git_aggregator.repo  odoo   Start aggregation of /opt/odoo/custom/src/odoo
(INFO) [13:20:01] git_aggregator.repo  odoo   Switch to branch saas-12.5
Reset branch 'saas-12.5'
(INFO) [13:20:03] git_aggregator.repo  odoo   Remote already exists ocb <https://github.com/OCA/OCB.git>
(INFO) [13:20:03] git_aggregator.repo  odoo   Remote already exists odoo <https://github.com/odoo/odoo.git>
(INFO) [13:20:03] git_aggregator.repo  odoo   Remote already exists openupgrade <https://github.com/OCA/OpenUpgrade.git>
(INFO) [13:20:03] git_aggregator.repo  odoo   Fetching required remotes
fatal: Couldn't find remote ref saas-12.5
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/git_aggregator/main.py", line 204, in aggregate_repo
    repo.aggregate()
  File "/usr/local/lib/python3.7/site-packages/git_aggregator/repo.py", line 179, in aggregate
    self.fetch()
  File "/usr/local/lib/python3.7/site-packages/git_aggregator/repo.py", line 202, in fetch
    self.log_call(cmd, cwd=self.cwd)
  File "/usr/local/lib/python3.7/site-packages/git_aggregator/repo.py", line 159, in log_call
    ret = callwith(cmd, **kw)
  File "/usr/local/lib/python3.7/subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '('git', 'fetch', '--depth', '100', 'ocb', 'saas-12.5')' returned non-zero exit status 128.
Traceback (most recent call last):
  File "/usr/local/bin/autoaggregate", line 127, in <module>
    aggregate(REPOS_YAML)
  File "/usr/local/bin/autoaggregate", line 46, in aggregate
    stdout=sys.stdout,
  File "/usr/local/lib/python3.7/subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['gitaggregate', '--expand-env', '--config', '/opt/odoo/custom/src/repos.yaml', '--log-level', 'INFO', '--jobs', '4']' returned non-zero exit status 1.
```
on docker-compose -f setup-devel.yaml run --rm odoo
it seems that the variables are not correctly passed in the build context. Adding the last line to
ODOO_VERSION: $ODOO_MINOR to setup-devel.yaml seems to fix that.

```
environment:
            DEPTH_DEFAULT: 100
            # XXX Export these variables before running setup to own the files
            UID: "${UID:-1000}"
            GID: "${GID:-1000}"
            UMASK: "$UMASK"
            ODOO_VERSION: $ODOO_MINOR
```

**Comment by OSevangelist** [comment]
@Yajo missed you in OXP2019 man. Do you have an idea why this only works by setting the environment variable explicitly ?

**Comment by pedrobaeza** [comment]
Have you tried with latest image? I think this was fixed.

**Comment by OSevangelist** [comment]
I think yes but i'll try it tomorrow again and report back

**Comment by yajo** [comment]
Hi there @OSevangelist, thanks! I hope to go next year :wink: 

Regarding the issue, the main point is:

```
fatal: Couldn't find remote ref saas-12.5
```

That's because I had to hack a bit to get the pre-release working. Now that `13.0` branches exist, I merged https://github.com/Tecnativa/doodba/pull/244 and it's released already, so just update as @pedrobaeza said and it should be OK.

Optimistically closing.
